ci/scripts/deterministic-e2e-test.sh

```diff
@@ -9,26 +9,40 @@ echo "--- Download artifacts"
 buildkite-agent artifact download risingwave_simulation .
 chmod +x ./risingwave_simulation
 
-export RUNNER=./risingwave_simulation
-export RUST_LOG=off
+export RUST_LOG=info
+export LOGDIR=.risingwave/log
 
 echo "--- deterministic simulation e2e, ci-3cn-1fe, ddl"
-seq 16 | parallel MADSIM_TEST_SEED={} $RUNNER './e2e_test/ddl/\*\*/\*.slt'
+mkdir -p $LOGDIR
```

Comment:
I think we can add the name here.

`$LOGDIR/deterministic-simulation-e2e-ci-3cn-1fe-ddl.log`